kinetic/release-build.yaml

```diff
@@ -0,0 +1,62 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+jenkins_binary_job_priority: 81
```

Comment:
This priority collides with Indigo source jobs. I think the numbers need to be updated for the other distros to ensure that all source jobs have a higher priority then all binary jobs as well as in each job category kinetic is higher then jade is higher then indigo.